### ISSUE TITLE
Bricked

### ISSUE THREAD

**Issue by AndyPook** [JamesSwift/localbytes-plug-pm#10]
Hi,

I am trying to "upgrade" some 2023-03 plugs flashed with tasmota to this esp firmware.
One worked fine. The second seems to be "bricked". I may have been a little impatient while going through the tasmota-minimum, localbytes-plug-pm cycle.

Whatever happened, the plug is now unresponsive. Button does nothing, led never lights up.
It does seem to appear on th previous configured network (unifi shows it is connected and it's IP). Browsing to that IP returns ERR_CONNECTION_REFUSED.

I have tried the  tasmota power cycle, to no avail. I am assuming that the last firmware upload failed part way.

The question...
I am familiar with esp dev boards. Is there a way to get physically into these plugs so that I can flash it directly?
They appear to be glued closed. Is there a trick to opening them?

Cheers,

**Comment by AndyPook** [comment]
TL;DR: I have been able to recover it. So it is fully working now with esphome.

I could see it appearing on the network (ip address, pingable etc) but with connection refused from a browser.
I have been running ESPHome-beta (for other reasons) and recently updated. 
I thought I'd retry the plug. As soon as I powered it on it appeared for "configuration" in HA. Then for adoption in esphome. Followed through and re-installed.

Although the web-ui was inaccessible, there must have been enough of the firmware running to enable discovery and flashing via wifi,

I don't know if it was the new esphome-beta or the long period of power off... but all is now good.

**Comment by JamesSwift** [comment]
Hi. Glad you've been able to recover the plug. Sorry I've only just seen this issue, I've been away from home for some time.
